src/devices/alecto.c

```diff
@@ -89,7 +89,7 @@ static int alectov1_callback(bitbuffer_t *bitbuffer) {
         /* Quit if checksup does not work out */
         if (csum != (bb[1][4] >> 4) || csum2 != (bb[5][4] >> 4)) {
             //fprintf(stdout, "\nAlectoV1 CRC error");
-            if(debug_output) {
+            if(debug_output >= 1) {
```

Comment:
In my opinion this change should be deleted.   This isn't overly verbose, so it seems appropriate for debug_level == 1.